contribute/style-guides/e2e-plugins.md

```diff
@@ -1,16 +1,18 @@
+Grafana provides the [`@grafana/plugin-e2e`](https://www.npmjs.com/package/@grafana/plugin-e2e?activeTab=readme) tool for testing end-to-end Grafana plugins. The `@grafana/plugin-e2e` tool extends [`@playwright/test`](https://playwright.dev/) capabilities with relevant fixtures, models, and expect matchers to simplify plugin development. 
 
-## Adding end-to-end tests for a core plugin
+You can use the tool to enable comprehensive end-to-end testing of Grafana plugins across multiple versions of Grafana. For information on how to get started with plugin end-to-end testing and Playwright, refer to the [E2E Get started](https://grafana.com/developers/plugin-tools/e2e-test-a-plugin/get-started) guide.
 
-Playwright end-to-end tests for plugins should be added to the [`e2e/plugin-e2e`](https://github.com/grafana/grafana/tree/main/e2e/plugin-e2e) directory.
+## Add end-to-end tests for a core plugin
 
-1. Add a new directory that has the name as your plugin [`here`](https://github.com/grafana/grafana/tree/main/e2e/plugin-e2e). This is where your plugin tests will be kept.
+Playwright end-to-end tests for plugins should be added to the [`e2e/plugin-e2e`](https://github.com/grafana/grafana/tree/main/e2e/plugin-e2e) directory in a specific _project_. In Playwright's terminology, [projects](https://playwright.dev/docs/test-projects) are logical groupings of tests. All tests in a Playwright project share the same configuration.
 
-2. Playwright uses [projects](https://playwright.dev/docs/test-projects) to logically group tests together. All tests in a project share the same configuration.
-   In the [Playwright config file](https://github.com/grafana/grafana/blob/main/playwright.config.ts), add a new project item. Make sure the `name` and the `testDir` sub directory matches the name of the directory that contains your plugin tests.
-   Adding `'authenticate'` to the list of dependencies and specifying `'playwright/.auth/admin.json'` as storage state will ensure all tests in your project will start already authenticated as an admin user. If you wish to use a different role for and perhaps test RBAC for some of your tests, please refer to the plugin-e2e [documentation](https://grafana.com/developers/plugin-tools/e2e-test-a-plugin/use-authentication).
+1. Add a new directory with the same name as your plugin at [`https://github.com/grafana/grafana/tree/main/e2e/plugin-e2e`](https://github.com/grafana/grafana/tree/main/e2e/plugin-e2e). This is where your plugin tests are kept.
+
+1. In the [Playwright configuration file](https://github.com/grafana/grafana/blob/main/playwright.config.ts), add a new project item. Make sure the `name` and the `testDir` subdirectory matches the name of the directory that contains your plugin tests.
```

Comment:
I think there might be an agreement issue in this sentence. Is it `match` instead of `matches`?

Reply:
done